.github/workflows/common-slack-notify-submitted.yml

```
@@ -0,0 +1,61 @@
+
+          SLACK_REVIEWER_ID=${GITHUB_TO_SLACK[$PR_REVIEWER]:-$PR_REVIEWER}
+          SLACK_AUTHOR_ID=${GITHUB_TO_SLACK[$PR_AUTHOR]:-$PR_AUTHOR}
+
+          SLACK_REVIEWER_MENTION="<@$SLACK_REVIEWER_ID>"
+          SLACK_AUTHOR_MENTION="<@$SLACK_AUTHOR_ID>"
+
+          if [[ "$PR_REVIEWER" == *"[bot]" ]]; then
+            echo "Bot 계정 리뷰 요청은 알림 생략"
+            exit 0
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)